### PR TITLE
feat(jupyterlab): bump main images to 3.0

### LINKED ISSUE #219: upgrade jupyterlab 2 images to jupyterlab 3
use Jupyterlab 3.0 for production images in kubeflow-containers and update manifest to use these.

Note: This is different from the OL-compliant images, some extensions such as jupyterlab-git and jupyterlab-lsp (#218)  that are not OL-compliant will be included.

Eventually we will only have the OL-compliant images once i18n is available for all our desired extensions

### PR DESCRIPTION
**Pull request by JessicaBarh**
Resolves #219

### CODE REVIEW

File: docker-bits/6_JupyterLab.Dockerfile [hunk 1]
```diff
@@ -79,6 +70,10 @@ RUN pip install --quiet \
   fix-permissions $CONDA_DIR && \
   fix-permissions /home/$NB_USER
 
+# Install R and Julia language servers
+RUN julia -e 'using Pkg; Pkg.add("LanguageServer")' && \
+    conda install -c conda-forge r-languageserver
```
Comment (blairdrummond) [inline docker-bits/6_JupyterLab.Dockerfile:75]: Do we need to clean the layer and/or fix permissions after this?

Comment (ca-scribner) [reply docker-bits/6_JupyterLab.Dockerfile:75]: Agreed, this will need a fix permissions and a conda clean or whatever the command is to clear the cache

Comment (JessicaBarh) [reply docker-bits/6_JupyterLab.Dockerfile:75]: Sounds good, I'll add that in.

**Review by ca-scribner — APPROVED** [review]
lgtm!

**Pull request merged**